app/helpers/application_helper.rb

```diff
@@ -545,14 +545,19 @@ def user_set?(field)
     params[:host] && params[:host][:hostgroup_id] && params[:host][field]
   end
 
-  def notifications
-    content_tag :div, :id => 'notifications', :'data-flash' => flash_notifiations.to_json.html_safe do
-      mount_react_component('ToastNotifications', '#notifications')
+  def render_toast_notifications
```

Comment:
Why the change to `render_toast_notifications`?  This will break all plugins that use `<%= notifications %>`.

Reply:
I didn't really have a good reason to do so, only because I thought it is a better name.
Will migrate back.